### PR TITLE
Add 'Search for Dom Cobb' as a new option.

### PR DESCRIPTION
**Pull request by chellabeatrixkiddo**
Please review this, @casey-collab

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/search-for-dom-cobb/search-for-dom-cobb.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+You realize you came as a 'tourist' into Dom Cob's dream! Why does he have marshmallows for walls?
```
Comment (casey-collab) [inline english/search-for-dom-cobb/search-for-dom-cobb.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
It looks like the file `english/search-for-dom-cobb/.DS_Store` is empty.  Please make sure you don't add empty files to the repository.  You could add one sentence to this file, or delete the option that leads to it.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**